seeds/chainlink/chainlink_get_price_seed.csv

```diff
@@ -0,0 +1,19 @@
+blockchain,block_time,block_number,feed_name,oracle_price,proxy_address,aggregator_address,underlying_token_address
+bnb,2021-12-24 09:14,13750620,EUR / USD,1.133761,0x0bf79f617988c472dca68ff41efe1338955b9a80,0xd2528b74ca91bb07b9bd9685ce533367c6fa657c,0x23b8683ff98f9e4781552dfe6f12aa32814924e8
+bnb,2022-05-26 02:35,18126071,CHF / USD,1.0400741,0x964261740356cb4aad0c3d2003ce808a4176a46d,0x6c9c9757f0478bc38bf73abda27ac42864de0645,0x7c869b5a294b1314e985283d01c702b62224a05f
+bnb,2022-04-14 14:24,16941036,BRL / USD,0.212147,0x5cb1cb3ea5fb46de1ce1d0f3badb3212e8d8ef48,0x2f92dc0711ada3dc255e2197e7c15c8adc6b6537,0x316622977073bbc3df32e7d2a9b3c77596a0a603
+bnb,2022-07-02 23:45,19211655,ZAR / USD,0.060792,0xde1952a1bf53f8e558cc761ad2564884e55b2c6f,0x86eb1bb8c66f365ea3df12a565a46cea204f6283,0x6b8b9ae0627a7622c593a1696859ca753c61a670
+bnb,2022-07-22 12:24,19773003,GBP / USD,1.19700499,0x8faf16f710003e538189334541f5d4a391da46a0,0xe0a34b8fc5e80c877fd568bd22b49e1bca977b6f,0x048e9b1ddf9ebbb224812372280e94ccac443f9e
+polygon,2022-11-23 03:10,35950884,GBP / USD,1.18855,0x099a2540848573e94fb1ca0fa420b00acbbc845a,0x3f7f90e0f782e325401f6323ba93e717f519f382,0x767058f11800fba6a682e73a6e79ec5eb74fac8c
+polygon,2022-11-23 03:05,35950735,AUD / USD,0.66465,0x062df9c4efd2030e243ffcc398b652e8b8f95c6f,0x0a9823c5cd84099fde8566a1adf0f2bb41cc6e7d,0xcb7f1ef7246d1497b985f7fc45a1a31f04346133
+polygon,2022-11-23 03:11,35950885,CAD / USD,0.747468,0xaca44abb8b04d07d883202f99fa5e3c53ed57fb5,0x88b79bfce730bbb74f23ab8940b37b86859caa2e,0x8ca194a3b22077359b5732de53373d4afc11dee3
+polygon,2022-11-17 02:07,35701896,SGD / USD,0.72890016,0x8ce3cac0e6635ce04783709ca3cc4f5fc5304299,0x45ede0ea5cbbe380c663c7c3015cc7c986669fec,0xa926db7a4cc0cb1736d5ac60495ca8eb7214b503
+polygon,2021-07-30 10:55,17423257,CHF / USD,1.104948,0xc76f762cedf0f78a439727861628e0fdfe1e70c2,0x8123beacb5bca3afa0c9ff71b28549d58cec8176,0xbd1463f02f61676d53fd183c2b19282bff93d099
+polygon,2021-11-18 07:15,21507237,NZD / USD,0.70375,0xa302a0b8a499fd0f00449df0a490dede21105955,0xe63032a70f6eb617970829fbfa365d7c44bdbbbf,0x08e6d1f0c4877ef2993ad733fc6f1d022d0e9dbf
```

Comment:
Hi @0xroll, please have a look on this seed, as this record may not be available in the [query result](https://dune.com/queries/1707702?d=1)